charmcraft.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+type: charm
+parts:
+  charm:
+    charm-python-packages:
+      - setuptools  # for jinja2
```

Comment:
I remember the ones for `cryptography` are for pylxd, and the installation would fail without those.
`git` is needed for installing from git source for pylxd, and repo-policy-compliance.
`jinja2` is needed for the templating.
I think `cffi` is also for pylxd.

However, this was tested a while ago. Reviewing it again might be a good idea.

Reply:
Minor comments